RecoTracker/SpecialSeedGenerators/src/OutsideInMuonSeeder.cc

```diff
@@ -41,7 +37,7 @@
 #include "TrackingTools/PatternTools/interface/TrajectoryMeasurement.h"
 #include "TrackingTools/PatternTools/interface/TrajMeasLessEstim.h"
 
-class OutsideInMuonSeeder : public edm::stream::EDProducer<> {
+class OutsideInMuonSeeder final : public edm::stream::EDProducer<> {
```

Comment:
in pre C++11 virtual in derived class was a sort of documentation for overridden methods.
In a final class virtual is almost equivalent to override (the latter being of course preferred).
it will trigger compilation error if the method is not an overridden one.